pkgs/development/python-modules/aiohttp-remotes/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, aiohttp }:
+
+buildPythonPackage rec {
+  pname = "aiohttp_remotes";
+  version = "0.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "43c3f7e1c5ba27f29fb4dbde5d43b900b5b5fc7e37bf7e35e6eaedabaec4a3fc";
+  };
+
+  propagatedBuildInputs = [ aiohttp ];
+
+  # Tests not included in PyPI tarball.
+  # https://github.com/aio-libs/aiohttp-remotes/pull/57
+  doCheck = false;
```

Comment:
It seems like I was wrong and tests are already included. So please add an appropriate `checkPhase`.

Reply:
48 tests fail like this:

```
file /private/var/folders/fr/ch_phzr9785fp_c4dkgh7cc80000gp/T/tmp.My4OXIGFwV/aiohttp_remotes-0.1.2/tests/test_x_forwarded.py, line 176
  async def test_x_forwarded_strict_too_many_for(test_client):
      async def handler(request):
          return web.Response()

      app = web.Application()
      app.router.add_get('/', handler)
      await _setup(app, XForwardedStrict([['127.0.0.1']]))
      cl = await test_client(app)
      resp = await cl.get('/',
                          headers={'X-Forwarded-For':
                                   '10.10.10.10, 11.11.11.11'})
      assert resp.status == 400
E       fixture 'test_client' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, monkeypatch, pytestconfig, record_property, record_xml_attribute, record_xml_property, recwarn, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.
```

I suspect this was fixed by https://github.com/aio-libs/aiohttp-remotes/commit/cbaac2cad60be2ada2688a662cce40bbd6dd0dd0, which has not yet made it into a release.

Reply:
You can use `fetchpatch` to apply that patch.

Reply:
That works. I could just use fetchFromGitHub, since it was the commit immediately after the release. That might make it more obvious that it can be dropped whenever the next version is released.

Reply:
I'd still vote for using `fetchpatch`. It will make it more obvious that we're shipping the specified version. The patch will need to be removed after an update anyway because it wouldn't apply anymore afterwards which would lead to doh-proxy not building anymore.

Reply:
Turns out it doesn't fix it anyway. I'll keep at it and see if I can figure out what's wrong.

Reply:
Fixed :) Needed to add another `checkInput`, and use a different test deprecation fix patch.